tests/test_fsclient.py

```diff
@@ -134,9 +138,30 @@ def test_delete_one():
     pass
 
 
-@pytest.mark.skip("Not written")
 def test_find_one():
-    pass
+    client = FileSystemClient(rc)
+    cif_path1 = "../docs/examples/cifs/calculated/bs0018IIIsup4.rtv.simulated.cif"
```

Comment:
this is brittle and bound to break.  copy-paste the content into a variable in a python module and import it.  Call the module something like testing_cif_inputs.   Then use TempDirectory to write them to a tempfile.